src/System.IO.MemoryMappedFiles/src/System.IO.MemoryMappedFiles.csproj

```diff
@@ -13,12 +13,15 @@
     <ResourcesSourceOutputDirectory Condition="'$(TargetGroup)' == 'net46'">None</ResourcesSourceOutputDirectory>
     <PackageTargetFramework Condition="'$(TargetGroup)' == ''">netstandard1.3</PackageTargetFramework>
     <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.3</NuGetTargetMoniker>
+    <DefineConstants Condition="'$(TargetGroup)' == 'netcore50'">NETCORE50</DefineConstants>
```

Comment:
Maybe I missed it but where is this define used? If you do use it you should be sure to append it to the existing DefineContants list so you don't drop the common ones.